examples/data-sources/backupschedule/data_source_backup_schedule.tf

```diff
@@ -1,9 +1,11 @@
 data "tanzu-mission-control_backup_schedule" "demo" {
   scope {
-    management_cluster_name = "MGMT_CLS_NAME"
-    provisioner_name        = "PROVISIONER_NAME"
-    cluster_name            = "CLS_NAME"
-    name                    = "TARGET_LOCATION_NAME"
+    cluster {
+      management_cluster_name = "MGMT_CLS_NAME"
+      provisioner_name        = "PROVISIONER_NAME"
+      cluster_name            = "CLS_NAME"
+      name                    = "TARGET_LOCATION_NAME"
```

Comment:
```suggestion
      name                    = "BACKUP_SCHEDULE_NAME"
```

Reply:
TARGET_LOCATION_NAME is as per the UI terminology, better to stick with the same

Reply:
Done